docs/fsharp/using-fsharp-on-azure/blob-storage.md

```diff
@@ -21,7 +21,13 @@ To use this guide, you must first [create an Azure storage account](/azure/stora
 
 The samples in this article can be used in either an F# application or an F# script. To create an F# script, create a file with the `.fsx` extension, for example `blobs.fsx`, in your F# development environment.
 
-Next, use a [package manager](package-management.md) such as [Paket](https://fsprojects.github.io/Paket/) or [NuGet](https://www.nuget.org/) to install the `Azure.Storage.Blobs` and `Azure.Storage.Blobs.Models` and `Azure.Storage.Blobs.Specialized` packages and reference `Azure.Storage.Blobs.dll` in your script using a `#r` directive.
+### How to execute scripts
+
+Highlight a line and press `ALT + ENTER` to execute it section by section, the results are shown in the FSI console.
```

Comment:
I think that we should prefer discussing using `dotnet fsi` from the command line here;  this behavior is specific to certain editors, like Visual Studio, which we should not assume readers have access to.